src/components/Expandable/__stories__/Controlled.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { DecoratorFunction } from '@storybook/addons'
+import { useState } from 'react'
+import Expandable from '..'
+import Button from '../../Button'
+import Stack from '../../Stack'
+
+export const Controlled = () => {
```

Comment:
```suggestion
export const Controlled: Story = () => {
```

Then you can drop your as DecoratorFunction below ?